packages/rpc-graphql/src/resolvers/account.ts

```diff
@@ -3,29 +3,35 @@ import { GraphQLResolveInfo } from 'graphql';
 
 import { RpcGraphQLContext } from '../context';
 import { AccountLoaderArgs } from '../loaders/account';
+import { transformParsedAccountData } from '../loaders/transformers/account';
 
 export const resolveAccount = (fieldName: string) => {
     return (
         parent: { [x: string]: Address },
         args: AccountLoaderArgs,
         context: RpcGraphQLContext,
         info: GraphQLResolveInfo | undefined,
-    ) =>
-        parent[fieldName] === null ? null : context.loaders.account.load({ ...args, address: parent[fieldName] }, info);
+    ) => {
+        return parent[fieldName] == null
+            ? null
+            : context.loaders.account.load({ ...args, address: parent[fieldName] }, info);
+    };
 };
 
 export const resolveAccountData = () => {
-    return (
+    return async (
         parent: { address: Address },
         args: { encoding: AccountLoaderArgs['encoding']; dataSlice?: AccountLoaderArgs['dataSlice'] },
         context: RpcGraphQLContext,
         info: GraphQLResolveInfo | undefined,
     ) => {
-        return (
-            context.loaders.account
-                .load({ ...args, address: parent.address }, info)
-                // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                .then((account: any) => account.data)
-        );
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const account: any = await context.loaders.account.load({ ...args, address: parent.address }, info);
+        const data = account.data;
+        if (data.parsed) {
+            return transformParsedAccountData(data);
+        } else {
+            return data[0] === '' ? null : data[0];
+        }
```

Comment:
Slick. The only reason the transformer was so convoluted was to also return `encoding` to let `__resolveType` know whether or not to return a `AccountBase58`, `AccountBase64`, or `AccountBase64Zstd`, but those are history!